tensilelite/Tensile/KernelWriter.py

```diff
@@ -2622,6 +2622,14 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
         module.add(self.removeStagger(kernel, tensorParametersA))
         module.add(self.removeStagger(kernel, tensorParametersB))
 
+      supportOpt = True
+      if (tuple(kernel["ISA"]) != (9, 4, 2)) or \
+         (kernel["ProblemType"]["Sparse"]) or \
+         (kernel["LocalSplitU"] > 1) or \
+         (kernel["WaveSeparateGlobalReadA"] != 0) or (kernel["WaveSeparateGlobalReadB"] != 0) or \
+         (kernel["DirectToVgprA"] or kernel["DirectToVgprB"]):
+        supportOpt = False
```

Comment:
calculate this in solutionstruct.py and store the result into a internal kernel parameter

Reply:
<img width="455" alt="{0ED21664-04A0-4E73-83D2-D628BA4B1E63}" src="https://github.com/user-attachments/assets/c90fd54e-ec88-471c-858e-f9e11619a38b">